frontend/src/components/Volunteer/volunteer.css

```diff
@@ -103,4 +103,7 @@ label {
   .form-control {
     width: 100%;
   }
+  .text-left {
```

Comment:
I think instead of having this be inside of a media query of `min-width: 768px`, I think we want this to be a `max-width: 768px ` since we only want it for mobile